index.html

```diff
@@ -2,9 +2,128 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Document</title>
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
   </head>
-  <body></body>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <main id="main">
+      <h1 id="title">Dr. Norman Borlaug</h1>
+      <p>The man who saved a billion lives</p>
+      <!-- image section -->
+      <div id="img-div">
+        <img
+          src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg"
+          id="image"
+          alt="image"
+          max-width="550px"
+          height="100%"
+        />
```

Comment:
Please make the image responsive and I think it would be better to use image link in CSS file

Reply:
Please ensure to always state explicit reasons why you requested a change; it might be obvious to you and not so to others.